test/unitTests.cpp

```diff
@@ -44,3 +46,82 @@ TEST(UnitTests, UnaryMinusTest) {
 TEST(UnitTests, SqrtTest) {
   EXPECT_DOUBLE_EQ(std::sqrt(2), (2 * meter2).sqrt().convert(meter));
 }
+
+TEST(UnitTests, AbsFreeFunction) {
+  EXPECT_EQ(abs(3.0_m), 3.0_m);
+  EXPECT_EQ(abs(-3.0_m), 3.0_m);
+}
+
+TEST(UnitTests, PowRootTest) {
+  EXPECT_DOUBLE_EQ((pow<2>(3_in)).convert(inch2), 9.0);
```

Comment:
I can't convince myself that this test passes. Wouldn't this call `std::pow(0.0762, 2)` which is definitely not 9 inches?

Reply:
The call to `.convert(inch2)` converts the result of pow (0.00581 m²) to square inches (9.0 in²)